src/slim-select/css_classes.ts

```diff
@@ -1,111 +1,111 @@
 export type CssClassesPartial = Partial<CssClasses>
```

Comment:
What actually changed in this file? I see indent removals but then not and it thinks the whole thing changed.

Reply:
I didn't run prettier when I submitted this file in the CSS override PR, the only thing that changed is the indentation